docs/dasharo-tools-suite/releases.md

```diff
@@ -5,298 +5,11 @@ for Dasharo Tools Suite.
 
 ## **Latest release**
 
-**Versions newer than `v1.2.11` are not published here.**{ .md-banner--warning }
-Instead, they can be found on `meta-dts` GitHub repository, inside `Releases`
+Latest release can be found on `meta-dts` GitHub repository, inside `Releases`
 tab. The button below will take you there there.
 
 [Latest release][Latest]{ .md-button .center .md-feedback }
 
 [Latest]: https://github.com/Dasharo/meta-dts/releases
 
 **Using the latest release is strongly recommended.**
```

Comment:
@DaniilKl lets also add info that using older versions is not recommended as it may lead to bricking device, e.g. by using version without checks on flash lock

see: https://github.com/dasharo/dasharo-issues/issues/489#issuecomment-2020276508

Reply:
@TomaszAIR, added here eb415381108dee5438e590f8fefbdfb89fe94838.

Reply:
@DaniilKl I would not say that it is a bug, IIRC locks were not present in previous Dasharo releases so we did not need checks.

Reply:
@TomaszAIR, amended here 6f21222db6935dbac821d1f0e308041ea84841af.